src/components/CButton/index.meta.js

```diff
@@ -3,6 +3,18 @@ export default {
   type: 'button',
   name: 'Button',
   icon: 'crop_landscape',
+  actions: [
+    {
+      name: 'setItemDisabledValue',
```

Comment:
Not sure where this method (`setItemDisabledValue`) is defined?

Reply:
Since most of the components can use this action and it's repeatable I've defined it inside Element component (as one of "common" methods).